vividus-soft-assert/src/main/java/org/vividus/softassert/ISoftAssert.java

```diff
@@ -31,6 +32,16 @@
      */
     boolean assertTrue(String description, boolean condition);
 
+    /**
+     * Asserts that a condition is true. If it isn't, an assertion error is added to the collection.
+     * Can be specified an additional operation which will always be executed.
```

Comment:
```suggestion
     * An additional operation which will always be executed is specified.
```

Reply:
also please add the description the operation is extremely useful when fail-fast is enabled